src/fiatExchanges/utils.tsx

```diff
@@ -144,48 +238,24 @@ export const fetchSimplexPaymentData = async (
   }
 }
 
-export const isSimplexQuote = (quote?: SimplexQuote | ProviderQuote): quote is SimplexQuote =>
-  !!quote && 'wallet_id' in quote
+export const isSimplexQuote = (
+  quote?: SimplexQuote | ProviderQuote | RawProviderQuote | RawSimplexQuote
+): quote is SimplexQuote => !!quote && 'wallet_id' in quote
 
-export const isProviderQuote = (quote?: SimplexQuote | ProviderQuote): quote is ProviderQuote =>
-  !!quote && 'returnedAmount' in quote
+export const isProviderQuote = (
+  quote?: SimplexQuote | ProviderQuote | RawProviderQuote | RawSimplexQuote
+): quote is ProviderQuote => !!quote && 'returnedAmount' in quote
 
-export const getLowestFeeValueFromQuotes = (quote?: SimplexQuote | ProviderQuote[]) => {
-  if (!quote) {
-    return
-  }
-
-  if (Array.isArray(quote)) {
-    if (quote.length > 1 && isProviderQuote(quote[0]) && isProviderQuote(quote[1])) {
-      return quote[0].fiatFee < quote[1].fiatFee ? quote[0].fiatFee : quote[1].fiatFee
-    } else if (isProviderQuote(quote[0])) {
-      return quote[0].fiatFee
-    }
-  } else if (isSimplexQuote(quote)) {
+export const getFeeValueFromQuotes = (quote?: SimplexQuote | ProviderQuote) => {
+  if (isSimplexQuote(quote)) {
     return quote.fiat_money.total_amount - quote.fiat_money.base_amount
   }
+  return quote?.fiatFee
 }
 
-// Leaving unoptimized for now because sorting is most relevant when fees will be visible
-export const sortProviders = (provider1: CicoProvider, provider2: CicoProvider) => {
-  if (provider1.unavailable) {
-    return 1
-  }
-
-  if (provider2.unavailable) {
-    return -1
-  }
-
-  if (provider1.restricted && !provider2.restricted) {
-    return 1
-  }
-
-  if (provider2.restricted && !provider1.restricted) {
-    return -1
-  }
-
-  const providerFee1 = getLowestFeeValueFromQuotes(provider1.quote)
-  const providerFee2 = getLowestFeeValueFromQuotes(provider2.quote)
+export const sortQuotes = ({ quote: quote1 }: CicoQuote, { quote: quote2 }: CicoQuote) => {
+  const providerFee1 = getFeeValueFromQuotes(quote1)
```

Comment:
```suggestion
  const providerFee1 = getFeeValueFromQuotes(quote1) ?? 0
  const providerFee2 = getFeeValueFromQuotes(quote2) ?? 0
  return providerFee1 - providerFee2
```
code golf (for this whole sort callback)